index.html

```diff
@@ -13,6 +13,8 @@
 
 <body>
 
+    <main>  
+        
     <nav class="top-nav">
         <ul>
             <li><i class="fa fa-apple" aria-hidden="true"></i></li>
```

Comment:
- [ ] The gradient is slightly different. Kindly fix that.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83212066-3c40dd80-a124-11ea-85a6-c5e9b4b10382.png) | ![image](https://user-images.githubusercontent.com/57414789/83212079-44008200-a124-11ea-96dc-dbe00e02c3cc.png)